xcape/core.py

```
@@ -129,40 +129,78 @@ def calc_cape(p, t, td, ps, ts, tds, **kwargs):
     zMUlev : array-like
         height of MUlev (m)
      """
-    
+    if len(args)>1:
+        print('too many optional arguments')
```

You want to raise an error here, not print.
```
raise ValueError("Too many optional arguments.")
```